src/kernel.jl

```diff
@@ -22,7 +22,7 @@ ENV["COLUMNS"] = get(ENV, "COLUMNS", 80)
 
 IJulia.init(ARGS)
 
-let startupfile = abspath(homedir(), ".julia", "config", "startup_ijulia.jl")
+let startupfile = abspath(DEPOT_PATH[1], "config", "startup_ijulia.jl")
```

Comment:
The whole thing should be wrapped in `!isempty(DEPOT_PATH)` similar to how [`startup.jl` is loaded](https://github.com/JuliaLang/julia/blob/5d02c59185ef31703891adc748e0ed414a9a5b3f/base/client.jl#L327)